mmv1/products/healthcare/go_FhirStore.yaml

```
@@ -0,0 +1,335 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Warning: This is a temporary file, and should not be edited directly
+---
+name: 'FhirStore'
+kind: 'healthcare#fhirStore'
+description: |
+  A FhirStore is a datastore inside a Healthcare dataset that conforms to the FHIR (https://www.hl7.org/fhir/STU3/)
+  standard for Healthcare information exchange
+references:
+  guides:
+    'Creating a FHIR store': 'https://cloud.google.com/healthcare/docs/how-tos/fhir'
+  api: 'https://cloud.google.com/healthcare/docs/reference/rest/v1/projects.locations.datasets.fhirStores'
+docs:
+id_format: '{{dataset}}/fhirStores/{{name}}'
+base_url: '{{dataset}}/fhirStores?fhirStoreId={{name}}'
+self_link: '{{dataset}}/fhirStores/{{name}}'
+update_verb: 'PATCH'
+update_mask: true
+import_format:
+  - '{{dataset}}/fhirStores/{{name}}'
+timeouts:
+  insert_minutes: 20
+  update_minutes: 20
+  delete_minutes: 20
+custom_code:
+  decoder: 'templates/terraform/decoders/go/long_name_to_self_link.go.tmpl'
+  custom_import: 'templates/terraform/custom_import/go/healthcare_fhir_store.go.tmpl'
+skip_sweeper: true
+examples:
+  - name: 'healthcare_fhir_store_basic'
+    primary_resource_id: 'default'
+    vars:
+      dataset_name: 'example-dataset'
+      fhir_store_name: 'example-fhir-store'
+      pubsub_topic: 'fhir-notifications'
+  - name: 'healthcare_fhir_store_streaming_config'
+    primary_resource_id: 'default'
+    vars:
+      dataset_name: 'example-dataset'
+      fhir_store_name: 'example-fhir-store'
+      pubsub_topic: 'fhir-notifications'
+      bq_dataset_name: 'bq_example_dataset'
+    test_vars_overrides:
+      'policyChanged': ' acctest.BootstrapPSARoles(t, "service-", "gcp-sa-healthcare", []string{"roles/bigquery.dataEditor", "roles/bigquery.jobUser"})'
+  - name: 'healthcare_fhir_store_notification_config'
+    primary_resource_id: 'default'
+    vars:
+      dataset_name: 'example-dataset'
+      fhir_store_name: 'example-fhir-store'
+      pubsub_topic: 'fhir-notifications'
+    skip_docs: true
```

oh I should edit these too, I'll pre emptively edit these then do another passover
```suggestion
    exclude_docs: true
```